docs/development.rst

```diff
@@ -319,8 +319,7 @@ is not compatible with production mode.
 To emulate production, and test compressed and hashed assets locally:
 
 #. Set the environment variable ``DEBUG=False``
-#. Start (``docker-compose up -d``) or restart (``docker-compose restart``)
-   your Docker services.
+#. Start (``docker-compose up -d``) your Docker services.
```

Comment:
It should be `docker-compose up --force-recreate`, because if existing container is running, it wont recreate them.

Reply:
Thanks for the advice! I've updated accordingly